ext/mbstring/tests/bug81298.phpt

```diff
@@ -1,20 +0,0 @@
---TEST--
-Bug #81298: mb_detect_encoding() segfaults when 7bit encoding is specified
---EXTENSIONS--
-mbstring
---FILE--
-<?php
-
-var_dump(mb_detect_encoding("foobar.", "7bit"));
-var_dump(mb_detect_encoding("foobar.", "7bit,ascii"));
-var_dump(mb_detect_encoding("foobar.", "7bit,ascii,utf8"));
-var_dump(mb_detect_encoding("foobar.", "html"));
-var_dump(mb_detect_encoding("foobar.", "ascii,html"));
```

Comment:
I'd either retain this test or extend the other one to show the `false` return value if only non-encodings are specified.